chpl/chpl-service/src/main/java/gov/healthit/chpl/domain/CertifiedProductSearchBasicDetails.java

```diff
@@ -252,221 +254,18 @@
      */
     private CertifiedProductSed sed;
 
+    /**
```

Comment:
I see you added a comment here to the new field. I thought our style guidance was more along the lines of remove comments when they are not telling you anything useful (i.e. most of the time). Do you want to clean up the comments in this class instead? I can see in the diff the one below for "Default Constructor" could probably be removed at least.